backend/app/model/top_container.rb

```diff
@@ -348,22 +348,24 @@ def self.bulk_update_location(ids, location_uri)
     begin
       relationship.handle_delete(relationship.find_by_participant_ids(TopContainer, ids).select{|v| v.status == 'current'}.map(&:id))
 
-      location = Location[JSONModel(:location).id_for(location_uri)]
+      unless location_uri.empty?
```

Comment:
@lorawoodford I wonder if you shouldn't perform a strip on location_uri in case someone uses the space bar to clear out the location causing the location_uri to be one or more blank spaces?

Reply:
@lmcglohon - As we discussed, this isn't necessary here, but I did go add it to the batch update that handles ILS holdings information as that text field could contain nothing but whitespace.